README.md

```diff
@@ -1,8 +1,21 @@
-# Nvidium
+# nvidium
```

Comment:
Why Change Nvidium to nvidium?

Reply:
#126, I think it looks cleaner and less harsh stylized that way